components/brave_rewards/resources/android_page/qrUtils.ts

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
this is not needed as we don't have add funds

Reply:
Removed.